src/reference/asciidoc/overview.adoc

```diff
@@ -216,7 +216,7 @@ IMPORTANT: Starting with _version 4.0_, the `@EnableIntegration` annotation has
 registration of Spring Integration infrastructure beans (see
 http://docs.spring.io/spring-integration/docs/latest-ga/api/org/springframework/integration/config/EnableIntegration.html[JavaDocs]).
 This annotation is required when only Java & Annotation configuration is used, e.g. with Spring Boot and/or
-Spring Integration Messaging Annotation support and Spring Integration Java DSL with no XML integration configuration.
+Spring Integration Messaging Annotations support and Spring Integration Java DSL with no XML integration configuration.
```

Comment:
Yeah... Not sure that I got the rule (maybe magic word `support`) and won't make similar mistake in the future, but thank you for catching that up, fixing and merging.